### PR TITLE
Initial commit, modules plus EventTransformationService

### PR DESCRIPTION
**Pull request by matt-rich**

### CODE REVIEW

File: stream-transformation-tool-service/src/main/java/uk/gov/justice/tools/eventsourcing/transformation/service/EventStreamTransformationService.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+/**
+ * Service to transform events contained on an event-stream.
+ */
+public class EventStreamTransformationService {
+
+    @Inject
+    EventSource eventSource;
+
+    Set<EventTransformation> transformations;
```
Comment (purple52) [inline stream-transformation-tool-service/src/main/java/uk/gov/justice/tools/eventsourcing/transformation/service/EventStreamTransformationService.java]: Is populating this another task?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>uk.gov.justice</groupId>
+        <artifactId>maven-framework-parent-pom</artifactId>
+        <version>1.0.0</version>
```
[the remark this answers is not in the document]
Comment (matt-rich) [reply pom.xml]: Done

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                <groupId>org.wildfly.swarm</groupId>
+                <artifactId>bom-all</artifactId>
+                <version>${wildfly.swarm.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency> -->
+            <dependency>
+                <groupId>uk.gov.justice</groupId>
+                <artifactId>maven-common-bom</artifactId>
+                <version>${common-bom.version}</version>
```
[the remark this answers is not in the document]
Comment (matt-rich) [reply pom.xml]: Done

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                <type>pom</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
```
[the remark this answers is not in the document]
Comment (matt-rich) [reply pom.xml]: Removed

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                <type>pom</type>
+            </dependency>
+            <dependency>
+                <groupId>uk.gov.justice.utils</groupId>
+                <artifactId>utilities-core</artifactId>
+                <version>${utilities.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jboss.ejb3</groupId>
+                <artifactId>jboss-ejb3-ext-api</artifactId>
```
[the remark this answers is not in the document]
Comment (matt-rich) [reply pom.xml]: Removed

File: stream-transformation-tool-service/src/main/java/uk/gov/justice/tools/eventsourcing/transformation/service/EventStreamTransformationService.java (hunk 1 above)
Comment (matt-rich) [reply stream-transformation-tool-service/src/main/java/uk/gov/justice/tools/eventsourcing/transformation/service/EventStreamTransformationService.java]: Yea, this needs to be made part of the Swarm app, with the sideloaded jar.

But we don't have that to integrate yet so I just made it work for the sake of testing.

Martin and I will do that integration next week when we turn this into the Swarm App.

File: stream-transformation-tool-api/pom.xml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>stream-transformation-tool-api</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>messaging-core</artifactId>
+            <version>${framework.version}</version>
```
[the remark this answers is not in the document]
Comment (matt-rich) [reply stream-transformation-tool-api/pom.xml]: Done, yea this one does, but the event-source is sadly not in the bom yet. We can put it in for fwk 3.0.0

**Pull request merged**